README.md

```diff
@@ -125,98 +125,97 @@ Creating a new open source project should almost always be a team decision. This
 - Will you build community around the project?
-   - Internal community: This is very important. The internal success of a project can give you a good idea about its potential to be adopted/used externally.
-   - External community: Who in your network would be willing to contribute?
-   - Sources
-       - LinkedIn groups
-       - community forums/boards
-       - Meetup audiences
-     - Will you need contributors with a different skillset to finish the project?
-     - Create a “TODO” in your README to invite contributions.
+  - Internal community: This is very important. The internal success of a project can give you a good idea about its potential to be adopted/used externally.
+  - External community: Who in your network would be willing to contribute?
+  - Sources
+    - LinkedIn groups
+    - Community forums/boards
+    - Meetup audiences
+    - Will you need contributors with a different skillset to finish the project?
+    - Create a “TODO” in your README to invite contributions.
 
 If you want to launch a new project separate from your team, talk to your delivery lead first. Let them know about your plan. Then ask yourself the questions immediately above.
 
-###The Main Zalando Repository: For the "Truly Open Source"
-Your code doesn’t have to be perfect before you open-source it. We *want* you to code in public from Day One. But to appear on our main org page, github.com/zalando, your project should be “truly open-source,” or in active development toward that goal. This means:
-- it’s useful and interesting to non-Zalandos. It might serve an entire language community, users of specific tools (Docker, Maven, etc.), or some other group.
-- it’s useable out of the box and free of Zalando dependencies
-- it has a clear README that:
-    - enables non-Zalando devs to download, install, run and use the project with minimal friction
-    - invites contributors via a TODO list
-    - clarifies if it's still in early development/a prototype
-- you respond promptly (within 72 hours) to PRs, issues, and queries
-
-####The Incubator: For the Open-Source, But Not "Truly Open Source"
-Projects that aren't "truly open source":
-- are dependent on our systems
-- are dependent on other Zalando open-source projects that haven't been widely adopted
-- are “experimental”/”maybe useful to others someday,” but you're not working on it (a Hack Week project, or something you've lost interest in)
-- lacking meaningful documentation (a README), and you're not working on any
-
-These are currently not suitable for the main org repo. Place them at github.com/zalando-incubator. The Incubator reiterates the Open Source First principle, "Share Your Code: All code shared between teams must be open source." Even if Incubator projects aren't currently useful outside of Zalando, we'd like to think that one day, they might be.
+###The Main Zalando Repository: For the “Truly Open Source”
```

Comment:
Sure, just make it the same everywhere ... though this separates real quotations in English from Strings in a given programming language